packages/@react-spectrum/dialog/stories/DialogTrigger.stories.tsx

```diff
@@ -355,6 +357,155 @@ storiesOf('DialogTrigger', module)
         </DialogTrigger>
       </Flex>
     )
+  )
+  .add(
+    'edges',
+    () => (
+      <Grid
+        areas={[
+          'top    top',
+          'start  end',
+          'bottom bottom'
+        ]}
+        columns={['auto', 'auto']}
+        rows={['size-1000', 'auto', 'size-1000']}
+        height="800px"
```

Comment:
would be good to increase this height so the story is scrollable so we can test the behavior for when the following button is scrolled to the very top/bottom of the page:
![Uploading image.png…]()